### PR TITLE
fix(memory): add stale preference injection regression fixtures

### LINKED ISSUE #1846: test(memory): add stale preference injection regression fixtures
## Description
Implement an LLM-friendly, independently shippable backlog item for **add stale preference injection regression fixtures**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- Memory storage, retrieval, injection, and redaction paths
- Project/profile scoped memory policy
- Memory audit and migration tooling

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add stale preference injection regression fixtures as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add stale preference injection regression fixtures is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add stale preference injection regression fixtures.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Persistent memory for agents**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- Adds dedicated stale preference memory injection regression fixtures for active-vs-stale preference ordering, archived injection-shaped memory, and tight-budget omission.
- Wires the fixture set into the LlmSkillHandler unit test so future memory prompt changes exercise the cases consistently.
- Documents how to interpret and extend the fixtures.

## Verification
- `npm ci`
- `npm run test --workspace @franken/orchestrator -- tests/unit/skills/llm-skill-handler.test.ts`
- `npm run build --workspace @franken/types`
- `npm run build --workspace @franken/observer`
- `npm run build --workspace @franken/brain`
- `npm run build --workspace @franken/critique`
- `npm run build --workspace @franken/governor`
- `npm run build --workspace @franken/planner`
- `npm run typecheck --workspace @franken/orchestrator`
- `npm run lint --workspace @franken/orchestrator` (passes with existing warnings)
- `npm run build --workspace @franken/orchestrator`

Closes #1846

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: packages/franken-orchestrator/tests/unit/skills/fixtures/stale-preference-memory-fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        `Stale user preference: ${'obsolete verbose reporting '.repeat(20)}`,
+        'User preference: report only actionable blockers.',
+      ],
+      knownErrors: ['Environment memory: Node 24 is supported in CI.'],
+    },
+    expectedActivePreference: 'User preference: report only actionable blockers.',
+    stalePreference: 'Stale user preference: obsolete verbose reporting',
+    expectedOmitted: ['obsolete verbose reporting obsolete verbose reporting'],
+    expectedPresent: ['[memory truncated:'],
+    memoryContextBudgetChars: 760,
```
[the remark this answers is not in the document]
Comment (djm204) [reply packages/franken-orchestrator/tests/unit/skills/fixtures/stale-preference-memory-fixtures.ts]: Fixed in a259f7d6. The tight-budget fixture now asserts that the current active preference and current ADR remain present, and the runtime now omits explicitly stale/archived entries instead of truncating them when they do not fit after higher-priority current facts. Re-ran the targeted LlmSkillHandler test, orchestrator typecheck/build, and orchestrator lint.

**Comment by djm204** [comment]
@codex review
